src/lightly_train/_constants.py

```diff
@@ -1,12 +0,0 @@
-#
-# Copyright (c) Lightly AG and affiliates.
-# All rights reserved.
-#
-# This source code is licensed under the license found in the
-# LICENSE file in the root directory of this source tree.
-#
-
-
-TIMM_PREFIX = "timm"
-TORCHVISION_PREFIX = "torchvision"
```

Comment:
TIMM_PREFIX and TORCHVISION_PREFIX are not used anymore. They are now part of the respective packages.